.env.test

```diff
@@ -22,4 +22,5 @@ AUTHZ_SERVER_HEALTH=health_endpoint
 AUTHZ_SERVER_VERIFICATION=verification_endpoint
 AUTHZ_SERVER_COMMUNITYCARE=community_care_endpoint
 AUTHZ_SERVER_CLAIMS=claims_endpoint
+AUTHZ_SERVER_CLINICAL_FHIR=clinical_fhir_endpoint
```

Comment:
added here to match with pre-existing code. We aren't using this but it's good to match the existing code for OAuth apis in case we need it later.

Reply:
good call